presto-docs/src/main/sphinx/release/release-0.206.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.206
+=============
+
+General Changes
+---------------
+
+* Fix query execution failure for certain queries containing a join followed by an aggregation
+  when ``dictionary_aggregation`` is enabled.
+* Fix query plan for aggregations where ``GROUP BY`` is present, but cardinality of the
```

Comment:
What does "fix query plan" mean? Did the queries fail at planning or runtime? Return wrong results?